docs/demo/generated/pandas.DataFrame.drop.rst

```diff
@@ -0,0 +1,6 @@
+pandas.DataFrame.drop
+=====================
+
+.. currentmodule:: pandas
+
+.. automethod:: DataFrame.drop
```

Comment:
It seems you accidentally added those to the tracked files? 
(we might need to add this directory to gitignore if that's not yet the case)

Reply:
Is is not yet part of gitignore. Should `docs/demo/generated` be added as a whole?

Reply:
Yes, this full /generated/ directory can be added to gitignore

Reply:
Ok check.